server/statistics/region_collection.go

```diff
@@ -163,6 +172,10 @@ func (r *RegionStatistics) Observe(region *core.RegionInfo, stores []*core.Store
 		PendingPeer: len(region.GetPendingPeers()) > 0,
 		LearnerPeer: len(region.GetLearners()) > 0,
 		EmptyRegion: region.GetApproximateSize() <= core.EmptyRegionApproximateSize,
+		SplitRegion: region.GetApproximateSize() >= int64(r.storeManager.GetStoreConfig().GetRegionMaxSize()) ||
+			region.GetApproximateKeys() >= int64(r.storeManager.GetStoreConfig().GetRegionMaxKeys()),
+		MergerRegion: region.GetApproximateSize() < int64(r.opt.GetScheduleConfig().MaxMergeRegionSize) &&
+			region.GetApproximateSize() < int64(r.opt.GetScheduleConfig().MaxMergeRegionKeys),
```

Comment:
It only takes the size/key into account, but there might be lots of factors that decide if a region needs to be merged. So the metrics of the output of `pd-ctl` may cause misleading.

Reply:
What about changing the name from `SplitRegion` and `MergerRegion` into `OversizeRegion` and `UndersizeRegion`? This should be more proper to the actual meaning.